Framework/Geometry/inc/MantidGeometry/Objects/MeshObject.h

```diff
@@ -160,6 +158,8 @@ class MANTID_GEOMETRY_DLL MeshObject : public IObject {
   void updateGeometryHandler();
 
 private:
+  /// Default constructor - should never be called
+  MeshObject();
```

Comment:
Delete it since it should never be called.

Reply:
I made it private to ensure this, else the compiler will create a default constructor that can be called.

Reply:
This is better, but as I just commented elsewhere, you can safely delete it as the compiler is already forbidden from creating one for you.

Reply:
Thanks for letting me know.